modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc

```diff
@@ -10,29 +10,39 @@ image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", titl
 image:logo-server-active.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
 image:logo-pcf-disabled.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
 
-The HTTP API analytics produced by applications running on the API gateway runtime can be easily stored in a database by configuring a few settings.
+You can configure Runtime Manager agent to store HTTP API analytics data from Mule apps running on the API gateway runtime in a database.
 
-[NOTE]
-====
-Sending data to third party software is a feature that is currently only available for applications deployed to local servers. It is not available for applications that you deploy via the Runtime Manager to xref:cloudhub.adoc[CloudHub]. See xref:deployment-strategies.adoc[Deployment Strategies].
-====
+Sending data to third-party software is only available for applications deployed to local servers.
+It is not available for applications that you deploy to CloudHub using Runtime Manager.
 
-To see how to send this same information to Splunk, ELK or other third party software, see xref:sending-data-from-arm-to-external-analytics-software.adoc[Sending data from Runtime Manager to External Analytics Software].
+For information about sending this data to Splunk, ELK, or another third-party software, see xref:sending-data-from-arm-to-external-analytics-software.adoc[Export Data to External Analytics Tools].
 
 == Prerequisites
 
-* API gateway standalone – version 2.1.0 or later
-* Runtime Manager agent plugin version 1.3.0 or later
+Ensure that the following software is installed:
+
+* API gateway standalone version 2.1.0 or later
+* Runtime Manager agent version 1.3.0 or later
 
-== Install the Runtime Manager Agent
+== Install or Update Runtime Manager Agent
 
-The Runtime Manager agent is in charge of sending messages out of Mule. To send event information to third party software, you must have version 1.3.0 or later of the Runtime Manager agent.
+Runtime Manager agent sends messages from Mule runtime engine.
+To send event information to third-party software, you must have version 1.3.0 or later of Runtime Manager agent.
 
-For help installing the agent, see xref:installing-and-configuring-runtime-manager-agent.adoc[Installing and Configuring Runtime Manager Agent].
+For information about updating the agent, see xref:installing-and-configuring-runtime-manager-agent.adoc[Install or Update Runtime Manager Agent].
 
 == Install API Analytics Tracking Database Internal Handlers
```

Comment:
This needs a hyphen or two, perhaps Install API Analytics-Tracking Database Internal Handlers?
Or rewording.